### ISSUE TITLE
can't modify variaible while debugging

### ISSUE THREAD

**Issue by NO-MEANINGS** [godotengine/godot#48716]
godot version: 3.3.stable.openSUSE
OS: openSUSE tumbleweed 20210511

I was trying to modify a variable while debugging, however the variable's value was not changed. I suspect there is a bug, or it is just a monitor for variables that is not for modify the variables' value? 

how to reproduce:
![bug](https://user-images.githubusercontent.com/52484447/118250266-dbe3b100-b4d8-11eb-80a0-0fca0f250b6b.gif)
use the example below and follow the gif
minimal example project:
[godotbug.tar.gz](https://github.com/godotengine/godot/files/6478112/godotbug.tar.gz)

**Comment by Zireael07** [comment]
I don't think you can change variables in that window, you have to use remote scene tree and change variables on the node, and if they're local to the function I don't think you can change them at all.

**Comment by josefkaragoli** [comment]
Your game is at a breakpoint meaning nothing is running. This is why changing variables doesn't update them; there's nothing to call for an update since the game is frozen

**Comment by NO-MEANINGS** [comment]
> I don't think you can change variables in that window, you have to use remote scene tree and change variables on the node, and if they're local to the function I don't think you can change them at all.

if it is not possible to change a variable that is local to a function, i think make them uneditable in the window could be better?

**Comment by NO-MEANINGS** [comment]
> Your game is at a breakpoint meaning nothing is running. This is why changing variables doesn't update them; there's nothing to call for an update since the game is frozen

if so, i have a question (sorry if my question is stupid, i am not a professional programmer XD
when the game is at a breakpoint. a property of a node can be changed. node‘s properties are actually variables which are same as the variable in my case. since it is possible to change variables which are properties of some nodes when the game is paused, why we can't change a more common variable? what makes the difference?

**Comment by josefkaragoli** [comment]
Nevermind I misunderstood what the issue was, you can't change those properties through there

**Comment by Faless** [comment]
Edited the title (because we usually call "inspector" the one in the side pane).
Yes, changing stack variables has no effect, so the local variables "inspector" should not allow editing.

**Comment by Faless** [comment]
That said, modifying a local object's property does work. So just setting it as disabled would probably remove that functionality.
Either we find a way to allow editing local variables, or we selectively disable them based on types.
